operations/finance.md

```diff
@@ -813,6 +813,72 @@ Creates payment for specified customer credit card and charges the credit card v
 | --- | --- | --- | --- |
 | PaymentId | string | required | Unique identifier of the payment [Accounting item](finance.md#accounting-item). |
 
+## Add tokenized credit card
+
+Allows you to attach tokenized credit card to the specific customer.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/creditCards/addTokenized`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "CustomerId": "e98995b0-140a-4208-bbeb-b77f2c43d6ee",
+    "CreditCardData": {
+        "PaymentStorageData": "2235057813",
+        "ObfuscatedNumber": "************1111",
```

Comment:
Is this correct obfuscatedNumber? Comparing with `41111********1111`? What is the difference?